test/file04.ts

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+import { Theme } from '@xsite-ui/common.style';
+
+export const Container = styled.div<{ theme?: Theme }>`
+  display: grid;
+  gap: ${({ theme }) => theme?.space?.[5] || 20}px;
```

Comment:
Using optional chaining (?.) with theme object. Possible runtime error if theme object is null or undefined. Consider performing nullish coalescing (??) check instead.